src/codecovio.jl

```diff
@@ -158,8 +158,8 @@ module Codecov
     a local git installation, rooted at `dir`. A repository token should be specified by a
     `token` keyword argument or the `CODECOV_TOKEN` environment variable.
     """
-    function submit_local(fcs::Vector{FileCoverage}, dir::AbstractString=joinpath(pwd(),".."); kwargs...)
-        LibGit2.with(LibGit2.GitRepo(dir)) do repo
+    function submit_local(fcs::Vector{FileCoverage}, dir::AbstractString=pwd(); kwargs...)
+        LibGit2.with(LibGit2.GitRepoExt(dir)) do repo
```

Comment:
Why `GitRepoExt`? I'm not very familiar with the difference. Does this create a git repo at the given directory if one doesn't already exist or something?

Reply:
No, it works in subdirectories. So if dir does not point a git repo, but just as a subdir of a git repo, GitRepoExt walks back the path hierarchy until it finds the root of the git repo.

This means that `GitRepoExt(pwd())` will work both when called from the root dir of the repo, and also when called from inside the test/ dir.

Anyway, this PR is not yet complete; I plan to resolve #159 and #160 with it, and perhaps some more.

Reply:
Okay, thanks.

Reply:
PR is now complete from my POV (i.e., modulo review change requests)